### PR TITLE
added python3 support and fix major issues

### LINKED ISSUE #91: Should exclude tabs.executeScript() calls to file:/// URL tabs
Originally reported here: https://bugs.chromium.org/p/chromium/issues/detail?id=602141

It seems the extension tries to inject script into tabs with file:/// URLs, ends up getting denied and produces console errors.

Couple of examples:
1)
extensions::lastError:133 Unchecked runtime.lastError while running tabs.executeScript: Cannot access contents of url "file:///Users/.../Test%20HTML%20File.htm". Extension manifest must request permission to access this host.
2)
extensions::lastError:133 Unchecked runtime.lastError while running tabs.executeScript: The extensions gallery cannot be scripted.
    at Object.callback (chrome-extension://noondiphcddnnabmjcihcjfbhfklnnep/background_compiled.js:38:330)

### PR DESCRIPTION
**Pull request by No-Cellist-7780**
added python3 support to test server and fixes #91 
Note : regex implementation not perfect suggestions are welcome

### CODE REVIEW

File: chrome/background.js [hunk 1]
```diff
@@ -401,11 +401,21 @@ passwordalert.background.injectContentScriptIntoAllTabs_ =
   chrome.tabs.query({}, function(tabs) {
     for (var i = 0; i < tabs.length; i++) {
       var tabIdentifier = tabs[i].id + ' - ' + tabs[i].url;
+
+      // Skip any pages not starting with https://
```
Comment (adhintz) [inline chrome/background.js:405]: I don't think we'd actually want to restrict to https:// nor https://www. as the code below appears to do.

I can't reproduce the issue at https://github.com/google/password-alert/issues/91 so there's probably nothing we need to do for this.

**Comment by adhintz** [comment]
Thanks for the contribution. Mind please reverting the change to background.js? Then I'll merge in the changes to test_server.py

File: chrome/background.js (hunk 1 above)
Comment (No-Cellist-7780) [reply chrome/background.js:405]: sure!

**Comment by adhintz** [comment]
Thanks for the contribution @No-Cellist-7780 !

**Pull request merged**